.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,3 +43,30 @@ Relevant software versions:
 ## Additional context
 
 <!-- Add any other context about the problem here, such as stack traces or debugging info. -->
+
+#### Definition of Done
+
+- [ ] This bug is **done**:
+  - [ ] Bug resolved to **user's** satisfaction
+  - [ ] Automated tests are passing
+  - [ ] Code is peer reviewed and pushed to master
+  - [ ] Deployed successfully to test environment
+  - [ ] Checked against CODING guidelines
+  - [ ] Relevant new tasks are added to backlog and communicated to the team
+  - [ ] Important decisions recorded in the issue ticket
+  - [ ] Sprint board is updated
+  - [ ] Readme/Changelog/Diagrams are updated
+  - [ ] Product Owner has approved as complete
+  - [ ] Meets non-functional requirements:
+    - [ ] Capacity (in total): At least 300TB of data and 100,000,000 files it total and ability to
+          grow 10% every year and a single dataset can contain 1,000,000 files and xx GB (TODO:
+          confirm).
+    - [ ] Capacity (per dataset): Up to 1TB and 150,000 files
+    - [ ] Cost: Data can be stored at < 0.5 NZD per GB per year (cost calculation
```

Comment:
Looks like there's some missing text here.

Reply:
Removed that bit.